src/lib/Graph.coffee

```diff
@@ -33,7 +34,7 @@ class Graph extends EventEmitter
   # and giving it a name:
   #
   #     myGraph = new Graph 'My very cool graph'
-  constructor: (@name = '') ->
+  constructor: (@name = '', @caseSensitive = false) ->
```

Comment:
Yeah I like that. I'll update soon. Should name stay it's own argument to preserve backwards compatibility?